src/cleaning_functions.py

```diff
@@ -24,26 +24,29 @@ def get_label_encoded_feature(feature):
     return [label_encoded_feature]
 
 
-def get_one_hot_encoded_feature(feature):
+def get_one_hot_encoded_feature(feature, feature_name):
```

Comment:
and its not violating design by single responsibility, its not like one output is calculating bayesian probability and the other is printing lists of hello kitten songs

Its just the encoding + the appropriate feature name or am I missing something ?